projects/plugins/social/src/js/components/admin-page/index.jsx

```diff
@@ -2,15 +2,17 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Button, ToggleControl } from '@wordpress/components';
 import { AdminPage, AdminSectionHero, Container, Col } from '@automattic/jetpack-components';
-import { useSelect } from '@wordpress/data';
-import { Button } from '@wordpress/components';
+import { useSelect, select as syncSelect, useDispatch } from '@wordpress/data';
 import { ConnectScreen, CONNECTION_STORE_ID } from '@automattic/jetpack-connection';
-import React from 'react';
+import React, { useEffect, useCallback } from 'react';
```

Comment:
I think we use `react` instead of `@wordpress/element` throughout the monorepo and only use later for `createInterpolateElement`

Reply:
Is that a historical thing from when we used React before the `@wordpress` packages, or is there an advantage in doing that?

I'm probably just thinking about this because previous JS development I've done in Jetpack has all been in the block editor.

Reply:
In my experience with Calypso and Jetpack (excluding the Block Editor), `@wordpress/*` doesn't get tossed around a whole lot. Far from an authoritative answer, but my guess is it's _mostly_ used in the Block Editor and not much otherwise, but I could be wrong on that.

Reply:
I'm happy to switch it to `@wordpress/*`, but if we make that switch we should perhaps also [update the starter plugin](https://github.com/Automattic/jetpack/blob/master/projects/plugins/starter-plugin/src/js/components/admin-page/index.jsx#L14). I'm not sure if that change has any side effects or if it will just keep working the same way.